src/views/editor/index.vue

```diff
@@ -569,6 +575,11 @@ export default {
           decode: await response.decode(args.fnReturnType),
           result: response.result
         })
+     
+       return this
+        .$store
+        .commit('terminal/createLine', `Result from call static: ${this.callStaticFn.staticResult.decode}`)
```

Comment:
@maptuhec can we keep this on a single line, even if it exceeds the column limit.

Reply:
@sadiqevani we can keep it, I was just following the "return" syntax that was used in the project.
if its better to have the expression on one line I will fix it

Reply:
@maptuhec I just would like to avoid having break lines like that and follow a more structured approach to code.

Looks good now. Thanks